app/lib/features/news/model/news_post_color_data.dart

```diff
@@ -1,12 +1,52 @@
-
 import 'package:flutter/material.dart';
 
-List<Color> newsPostColors = [
-  Colors.brown,
-  Colors.red,
-  Colors.orange,
-  Colors.deepPurple,
-  Colors.purple,
-  Colors.indigo,
-  Colors.pink,
+class PostColorScheme {
+  final Color backgroundColor;
+  final Color foregroundColor;
+  final Color linkColor;
+
+  const PostColorScheme({
+    required this.backgroundColor,
+    required this.foregroundColor,
+    required this.linkColor,
+  });
+}
+
+//https://www.google.com
+final List<PostColorScheme> postColorSchemes = [
+  PostColorScheme(
+    backgroundColor: const Color.fromARGB(255, 107, 65, 50),
+    foregroundColor: Colors.white,
+    linkColor: Colors.amberAccent,
```

Comment:
I assume you picked `amberAccess` because it has a strong contrast to the background colors? However, I have two issue with using that color:
1. _traditionally_ links are some shade of blue, not using blue makes it harder for people to recognise them, especially since we don't ever use that same color/pattern anywhere else. I'd really like to stick with blue for links
2. accessiblity guidelines recommend a contrast ratio of 3:1 between the link and the text color (so it easy to recognise especially for impaired people), but [amberAccent to white has a meager 1.39:1 ratio](https://webaim.org/resources/contrastchecker/?fcolor=FFD740&bcolor=FFFFFF).

If we intend to fix the colors for contrast we should not introduce a new contrast problem but fix it wholesale IMHO.